state/futures/selectors.ts

```diff
@@ -774,3 +774,18 @@ export const selectHasRemainingMargin = createSelector(
 			: posMargin.gt(0);
 	}
 );
+
+export const selectMaxUsdInputAmount = createSelector(
```

Comment:
Not really a suggestion for this PR, but this would be a great place to also add a check on OI caps to avoid users trying to submit orders over the cap and getting an error.

Reply:
Good call but I think we need some UX direction, ideally displaying to the user what the available OI is rather than a confusing max value. Will create a ticket.